packages/client/hmi-client/src/components/workflow/ops/calibrate-ensemble-ciemss/calibrate-ensemble-ciemss-operation.ts

```diff
@@ -87,12 +87,12 @@ export const CalibrateEnsembleCiemssOperation: Operation = {
 			configurationWeights: {},
 			timestampColName: '',
 			extra: {
-				solverMethod: speedPreset.method,
-				numParticles: speedPreset.numSamples,
-				numIterations: speedPreset.numIterations,
+				solverMethod: qualityPreset.method,
+				numParticles: qualityPreset.numSamples,
+				numIterations: qualityPreset.numIterations,
```

Comment:
Are there any adjustments needed to the polling interval and threshold with these changes?

Reply:
Hope not considering the user was expected to use this preset previously